static/webmention.js

```diff
@@ -202,9 +212,13 @@ A more detailed example:
         return text;
     }
 
+    function pluralise(word, count) {
+      // TODO: This pluralising depends on the language but assumes English
+      return t(word) + count > 1 ? 's' : '';
+    }
+
```

Comment:
Per the discussion on the i18n issue I think it would be better to just drop the count display entirely and just always show the plural form regardless of the number.

Reply:
Must have missed that. Will do!

Reply:
Yeah it was just an aside at https://github.com/PlaidWeb/webmention.js/issues/23#issuecomment-805170558